pkg/encoding/scheme.go

```diff
@@ -98,6 +98,7 @@ func init() {
 //
 // After this, RawExtensions in Kubernetes types will serialize kube-aggregator types
 // correctly.
+//nolint:errcheck
```

Comment:
We have a work in progress pr in relation to scheme so don't want to waste time sorting our error checking here for now so just added the `nolint:errcheck`